tests/spicy/types/unit/switch.spicy

```diff
@@ -4,6 +4,8 @@
 # @TEST-EXEC: echo 3234567890 | spicy-driver %INPUT.hlto >>output
 # @TEST-EXEC: echo 4234567890 | spicy-driver %INPUT.hlto >>output
 # @TEST-EXEC: btest-diff output
+#
+# @TEST-DOC: Checks unit switch statement.
```

Comment:
Want to add a test case for the error case where not all of size is being consumed?

Reply:
Did that and found an actual bug. 🤦